### ISSUE TITLE
popout's input field not initially focused

### ISSUE THREAD

**Issue by GoogleCodeExporter** [IIsi50MHz/chromey-calculator#159]
```
What steps will reproduce the problem?

Press the button or trigger it in some other way.  It automatically takes 
focus, requiring me to click in the text field every single time.  Sure, it's 
only one click, but it is still annoying.

What is the expected output?

What do you see instead?

What version of Chrome are you using?

What operating system are you using?

What country are you in?

Please provide any additional information below.

```

Original issue reported on code.google.com by `saumanah...@gmail.com` on 27 Sep 2012 at 9:31

**Comment by IIsi50MHz** [comment]
This behaviour was fist reported in issue #144 and identified as a Chrome bug. The Chrome bug seems to be fixed.

**Comment by IIsi50MHz** [comment]
This seems to still occur for some people, despite the Chrome bug being fixed. We need to find out what other circumstances can cause this (which is difficult because it hasn't happened for me in two or three years).

**Comment by IIsi50MHz** [comment]
possible fix in issue #247
